schema-derivatives-perpfutures.graphql

```diff
@@ -487,13 +492,11 @@ type UsageMetricsDailySnapshot @entity (immutable: true) {
 
   " Cumulative number of collateral Out from traders "
   cumulativeCollateralOut: Int!
-
   " Total number of pools "
   totalPoolCount: Int!
-
 }
 
-type UsageMetricsHourlySnapshot @entity (immutable: true) {
+type UsageMetricsHourlySnapshot @entity(immutable: false) {
```

Comment:
Instead of making them mutable, you can have an auxiliary entity that holds these counters. And then when taking the snapshot, copy it over from there and reset it.

And same for the daily ones

Reply:
fixed at 3864986b411c56a979d1bde345c6a0abe72c0402 Thanks!